README.md

```diff
@@ -17,9 +17,9 @@ git clone https://github.com/raprocks/spotidnldr.git spotidnldr
 
 Then change your working directory to the newly made folder named spotidnldr
 ```
-cd ./spotidnldr/spotidnldr
+cd spotidnldr
```

Comment:
Actually the command was correct sir just a step missing thats all please rewrite this as it was and i add this pull request